core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceMediaPlayer.java

```diff
@@ -322,7 +322,7 @@ protected void smartMarkAsPlayed(Playable media) {
 
         void onBufferingUpdate(int percent);
 
-        void reloadUI();
+        void reloadUI(boolean onlyMediaSession);
```

Comment:
Maybe I can break it into two calls, because the alternative name for the parameter would be notReloadUI (or reloadUI, or alsoReloadUI), which would be confusing with the method's name.